gatsby-node.js

```diff
@@ -34,7 +34,7 @@ exports.createPages = ({ graphql, boundActionCreators }) => {
           path: edge.node.slug, // required
           component: blogPost,
           context: {
-            slug: edge.node.slug,
+            slug: { eq: edge.node.slug },
```

Comment:
Ok and if not, we could probably get a patch accepted by Relay folks to make that restriction either optional or pluggable.